django_mri/models/managers/scan.py

```diff
@@ -21,3 +22,20 @@ def import_path(
     ) -> tuple:
         dicom_scans = self.import_dicom_data(path, progressbar, report)
         return {ScanType.DICOM.value: dicom_scans}
+
+    def scan_by_method(self, method: str) -> list:
```

Comment:
`by_method`? This seems to simply apply a `SequenceType` based filtering, doesn't it?

Reply:
It is, so how should we call it?

Reply:
I will leave it with this name for now, we can change it later if it's problematic.